mission-lotto/src/main/java/com/lotto/lottoTicket/service/exception/InsufficientBalanceException.java

```diff
@@ -0,0 +1,9 @@
+package com.lotto.lottoTicket.service.exception;
+
+public class InsufficientBalanceException extends IllegalStateException{
+
+    public InsufficientBalanceException() {
+        super("잔고가 부족합니다.");
+    }
+
+}
```

Comment:
다른 클래스도 마지막 개행이 아직도 추가가 되지 않습니다 ㅠ
```suggestion
}
```